content/blog/2020-07-22-july-20-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: July '20 Community Gems
+date: 2020-07-31
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  getting started with CML, configuring your DVC cache, and how to request a 
+  tutorial video.
+picture: 2020-07-31/Gems_July_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/july-20-community-gems/460
+tags:
+  - CML
```

Comment:
do we include DVC in gems?

Reply:
We don't usually have a DVC tag, since so far every gems has been DVC. Should we add tag?

Reply:
yeah, don't have a strong opinion ... we might have to start using DVC and CML (not only for gems) since we use the blog for two products